modules/graphql_content/src/Plugin/GraphQL/Fields/EntityById.php

```diff
@@ -19,12 +19,14 @@
  *   multi = false,
  *   weight = -1,
  *   arguments = {
- *     "id" = "String"
+ *     "id" = "Int",
+ *     "language" = "Language"
```

Comment:
Not every entity type is translatable, so you need to move this into the deriver and check if the current entity type you are deriving a plugin instance for is actually translatable.